servers/src/main/java/tachyon/worker/block/evictor/Evictor.java

```diff
@@ -17,11 +17,18 @@
 
 import tachyon.worker.BlockStoreLocation;
 import tachyon.worker.block.BlockMetaEventListener;
+import tachyon.worker.block.BlockMetadataManager;
 
 /**
  * Interface for the eviction policy in Tachyon
  */
-public interface Evictor {
+public abstract class Evictor {
```

Comment:
Is it better to be an abstract class , or an interface? I prefer interface if abstract class doesn't provide too much benefit over interface. In this particularly case, it seems the only benefit is to have a constructor, which requires to take only one parameter. We may want the construction scheme more extensible, by having a factory class/method for each type of Evictor?